packages/vm/tests/api/EIPs/eip-2315.spec.ts

```diff
@@ -3,16 +3,13 @@ import * as tape from 'tape'
 
 import { VM } from '../../../src/vm'
 
-import type { EVM } from '@ethereumjs/evm'
-
 tape('Berlin: EIP 2315 tests', (t) => {
   const common = new Common({ chain: Chain.Mainnet, hardfork: Hardfork.Berlin, eips: [2315] })
 
   const runTest = async function (test: any, st: tape.Test) {
     let i = 0
     const vm = await VM.create({ common })
-
-    ;(<EVM>vm.evm).on('step', function (step: any) {
+    vm.evm.events!.on('step', function (step: any) {
```

Comment:
Have updated all these unnecessary castings in the tests, should be good to go now. 🙂

Reply:
Yep this is much cleaner, thanks :smile: :+1: